1.8/overview/concepts.md

```diff
@@ -1,121 +1,282 @@
+- Examples: Most DC/OS components, (system) Marathon
 
-An offer represents available resources (e.g. cpu, disk, memory) which an agent reports to the master and the master offers to the registered frameworks in some order.
+#### <a name="system-service"></a>System Service
 
-# <a name="private"></a> Private agent node
+A system service is a service that implements or enhances the functionality of DC/OS itself, run as either a Marathon service or a systemd service, owned by the system (admin) user or DC/OS itself.
+- A system service may require special permissions to interact with other system services.
+- Permission to operate as a system service on an Enterprise DC/OS cluster requires specific fine-grained permissions, while on open DC/OS all logged in users have the same administrative permissions.
+- Examples: All DC/OS components
 
-Private agent nodes run DC/OS apps and services through a non-routable network that is only accessible from the admin zone or through the edgerouter from the public zone. By default DC/OS launches apps on private agent nodes. DC/OS agent nodes can be designated as [public][2] or [private][1] during installation. For more information, see the [Network Security documentation][11].
+#### <a name="user-service"></a>User Service
 
-# <a name="public"></a> Public agent node
+A user service is a Marathon service that is not a system service, owned by a user of the system.
 
-Public agent nodes run DC/OS apps and services in a publicly accessible network. DC/OS agent nodes can be designated as [public][2] or [private][1] during installation. For more information see:
+- This distinction is new and still evolving as namespacing is transformed into a system-wide first class pattern and mapped to fine-grained user and user group permissions.
+- Examples: Jenkins, Cassandra, Kafka, Tweeter
 
- - [Network Security][11]
- - [Creating a public agent node](/docs/1.8/administration/installing/custom/convert-agent-type/)
+### <a name="service-group"></a>Service Group
 
-# <a name="slave"></a>Slave
+A DC/OS service group is a hierarchical (path-like) set of DC/OS services for namespacing and organization.
 
-The synonym of slave node is worker or agent node. A Mesos agent node runs a discrete Mesos task on behalf of a framework. It is an agent instance registered with the Mesos master.
+- Service groups are currently only available for Marathon services, not systemd services.
+- This distinction may change as namespacing is transformed into a system-wide first class pattern.
 
-# <a name="stateabstraction"></a>State abstraction
+### <a name="job"></a>Job
 
-Mesos provides an abstraction for accessing storage for schedulers for Java and C++ only. This is the preferred method to access ZooKeeper for DC/OS services.
+A DC/OS job is a set of similar short-lived job instances, running as Mesos tasks, managed by the Jobs service scheduler (Metronome)
```

Comment:
uh, no. The name of the systemd unit is not the name of the component. I need to work on the components page...